code/game/objects/items/devices/instruments.dm

```diff
@@ -48,6 +51,34 @@
 	user.set_machine(src)
 	song.interact(user)
 
+/obj/item/instrument/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/musicaltuner))
+		var/mob/living/carbon/human/H = user
+		if (H.has_trait(TRAIT_MUSICIAN))
+			if (!tune_time)
+				H.visible_message("[H] tunes the [src] to perfection!", "<span class='notice'>You tune the [src] to perfection!</span>")
+				tune_time = 300
+				START_PROCESSING(SSobj, src)
+			else
+				to_chat(H, "<span class='notice'>[src] is already well tuned!</span>")
+		else
+			to_chat(H, "<span class='warning'>You have no idea how to use this.</span>")
+
+/obj/item/instrument/process()
+	if (tune_time && song.playing)
+		for (var/mob/living/M in song.hearing_mobs)
+			M.dizziness = max(0,M.dizziness-2)
+			M.jitteriness = max(0,M.jitteriness-2)
+			M.drowsyness = max(0,M.drowsyness-1)
+			M.confused = max(M.confused-1)
+			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "goodmusic", /datum/mood_event/goodmusic)
+			tune_time--

```

Comment:
Is this intentional to do it **per mob**?

Reply:
oh woops